app/src/main/java/io/homeassistant/companion/android/sensors/PhoneStateSensorManager.kt

```diff
@@ -41,16 +49,18 @@ class PhoneStateSensorManager : SensorManager {
 
     override val availableSensors: List<SensorManager.BasicSensor>
         get() = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP_MR1)
-            listOf(phoneState, sim_1, sim_2)
-        else listOf(phoneState)
+            listOf(phoneState, callNumber, sim_1, sim_2)
+        else listOf(phoneState, callNumber)
 
     override fun requiredPermissions(): Array<String> {
-        return arrayOf(Manifest.permission.READ_PHONE_STATE)
+        return arrayOf(Manifest.permission.READ_PHONE_STATE, Manifest.permission.READ_CALL_LOG)
```

Comment:
Does this mean that we can't use the phone state sensor without granting the `READ_CALL_LOG` permission?

Reply:
Yes for a while. I'm gonig to find simple solution to split required permissions per sensor entity. But it is not in the scope of current PR.

Reply:
Doesn't this mean it breaks existing functionality? I'm not sure thats a good thing for those that use it currently.

Reply:
It will just disable sensor. I'll try to fix this in next pr on Monday, so most likely only beta users can be affected. Or we can just merge this pr after it.

Reply:
Might need to move permissions to sensors rather than managers.